### PR TITLE
Add initial configuration for lit, spectrum and rollup

### PR DESCRIPTION
**Pull request by dylandepass**
TODO:

- [X] Setup Lit
- [X] Setup Spectrum
- [X] Setup Rollup Build
- [x] Setup Web Test Runner

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ node_modules/
 test-results.xml
 **/xcuserdata/*
 **/xcshareddata/*
+/dist
```
Comment (rofe) [inline .gitignore:11]: not sure about the leading slash...
```suggestion
dist
```

File: custom-elements.json [hunk 2]
```diff
@@ -0,0 +1,271 @@
+{
+  "schemaVersion": "1.0.0",
+  "readme": "",
+  "modules": [
+    {
```
Comment (rofe) [inline custom-elements.json:5]: Should we should exclude extension-related scripts and only include actual custom element modules?

File: src/extension/manifest.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
+{
+    "manifest_version": 3,
+    "name": "__MSG_title__",
+    "version": "6.35.2",
```
Comment (rofe) [inline src/extension/manifest.json:4]: Sync with `package.json`
```suggestion
    "version": "0.0.1",
```

**Review by rofe — APPROVED** [review]
lgtm, just a few comments

File: .gitignore (hunk 1 above)
Comment (dylandepass) [reply .gitignore:11]: Leading slash is valid but np, will fix to match everything else

File: custom-elements.json (hunk 2 above)
Comment (dylandepass) [reply custom-elements.json:5]: Fixed

File: .gitignore (hunk 1 above)
Comment (dylandepass) [reply .gitignore]: Fixed

**Pull request merged**